yarn.lock

```diff
@@ -0,0 +1,4854 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please do not check in lockfiles.
All dependency versions are locked by Nix, our JavaScript dependencies in `nix/node_modules.nix` specifically.
It is fine to use yarn locally, although this should not be necessary.
If you're using yarn because something in the Nix setup is broken, let me know.

Reply:
Ah I hadn't actually noticed this, you got a good point. I'll remove it from the PR